### ISSUE TITLE
Error when attempting to access packs page

### ISSUE THREAD

**Issue by InterstellarOne** [MehediH/winstall#130]
When attempting to access the packs page I get this error. 
`invalid json response body at https://api.winstall.app/packs reason: Unexpected token < in JSON at position 0`

**Comment by Riuaux** [comment]
Yeah, this happens on [https://winstall.app/packs](https://winstall.app/packs) but not for every `/packs/{id}` page; eg: [MS Visual C++ Redist & .NET | Riuaux](https://winstall.app/packs/3kTrqrW7d) does work, but [Work From Home](https://winstall.app/packs/A6JzO22Y1) (and all the other **Featured Packs** btw) returns a `Sorry! We could not load this pack.
Recieved error: [object Object]` response.

**Comment by DepressedOptimist** [comment]
Bump. Still happening as of today.

**Comment by st-jliu** [comment]
Dear InterstellarOne:

This project was taken over by Splashtop now, we will continue the development and maintenance, and we have re-design the whole packs feature.

You are welcome to provide further feedback and suggestions.

**Comment by jo-chemla** [comment]
Are packs now gated behind user signup/login? 
Originally you could access public pack page, that would be sad if this feature is now login-only.

**Comment by st-jliu** [comment]
User can still set pack as public
